Source/UI/NewUI/PaymentForm/Factory/Theme/BillingForm/Theme+BillingCountryInput.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Theme+BillingCountryInput.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 10/08/2022.
+//
+
+import UIKit
+
+public extension Theme {
+    
+    /// Theme generated Billing Country Input Style
+    struct ThemeBillingCountryInput: CellButtonStyle {
+        public var isMandatory: Bool = true
+        public var backgroundColor: UIColor = .clear
+        public var button: ElementButtonStyle
+        public var title: ElementStyle?
+        public var hint: ElementStyle?
+        public var mandatory: ElementStyle?
+        public var error: ElementErrorViewStyle?
+    }
+    
+    /// Create a Billing Country Input from provided themed components
+    func buildBillingCountryInput(button: CountryListButton,
+                                  title: ThemeTitle,
+                                  mandatory: ThemeMandatory? = nil,
+                                  subtitle: ThemeSubtitle? = nil,
+                                  error: ThemeError? = nil) -> ThemeBillingCountryInput {
+        ThemeBillingCountryInput(button: button,
+                                 title: title,
+                                 hint: subtitle,
+                                 mandatory: mandatory,
+                                 error: error)
+    }
+    
+    /// Create a Billing Country Input from provided content
+    func buildBillingCountryInput(buttonText: String,
+                                  buttonImage: UIImage? = nil,
+                                  title: String,
+                                  subtitle: String? = nil,
+                                  subtitleImage: UIImage? = nil,
+                                  isRequiredText: String? = nil,
+                                  errorText: String? = nil,
+                                  errorImage: UIImage? = nil) -> ThemeBillingCountryInput {
+        let subtitleText = subtitle ?? ""
+        let showSubtitle = !subtitleText.isEmpty || subtitleImage != nil
+        
+        let errorText = errorText ?? ""
+        let showError = !errorText.isEmpty || errorImage != nil
```

Comment:
Same. The names for booleans are usually in the form of a yes-or-no question, as if we are personally asking the boolean variable itself about its state.

Reply:
"Show Error?" can be a question. Whether you decide to phrases it as "Should Show", "May Show", "Can Show", you're only adding extra words...

Reply:
[Swift API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage)

> Uses of Boolean methods and properties should read as assertions about the receiver when the use is nonmutating, e.g. x.isEmpty, line1.intersects(line2).

Whilst we're looking at a local variable and not a property of an object, it would commonly be used as
```Swift
if showError
```

But once we apply that into the local use case, we would end up with code like
```Swift
var errorStyle: ErrorStyle?
if showError {
    errorStyle = makeErrorStyle()
}
```

Given we have 2-3 such properties these would stretch the code, so once we compact it we end up with what we have now:
```Swift
errorStyle: showError ? makeErrorStyle() : nil
```
making use of ternary operator as well as being safer and less verbose about the syntax. But once broken down it is the same thing.